### PR TITLE
Add NanoLimbo

### PR DESCRIPTION
**Pull request by mallusrgreatv2**
Info: https://github.com/Nan1t/NanoLimbo
#227

### CODE REVIEW

**Comment by mallusrgreatv2** [comment]
@LordRalex review?

File: minecraft-nanolimbo/minecraft-nanolimbo.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+  "name": "NanoLimbo",
```
Comment (LordRalex) [inline minecraft-nanolimbo/minecraft-nanolimbo.json:2]: Lowercase name

File: minecraft-nanolimbo/minecraft-nanolimbo.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    },
+    {
+      "files": [
+        "https://github.com/Nan1t/NanoLimbo/releases/download/v${version}/NanoLimbo-${version}-all.jar"
+      ],
+      "type": "download"
+    },
+    {
+      "source": "NanoLimbo-*-all.jar",
+      "target": "nanolimbo.jar",
```
Comment (LordRalex) [inline minecraft-nanolimbo/minecraft-nanolimbo.json:18]: To keep things consistent, can we call it server.jar?

**Comment by LordRalex** [comment]
Can you also create a -docker variant of this template? It will be removing the javadl, using just java as the command, and putting the image needed. You can refer to the other MC templates for an example.

**Pull request closed without merging**